nima/webclient/webclient/src/main/java/io/helidon/nima/webclient/ClientRequest.java

```diff
@@ -98,6 +99,32 @@ default B header(Http.HeaderName name, String value) {
      */
     B headers(Function<ClientRequestHeaders, WritableHeaders<?>> headersConsumer);
 
+    /**
```

Comment:
These two methods should be added in a separate PR.

Reply:
Okay, will move webclient files to separate PR